### PR TITLE
LICENSE and NOTICE changes for 0.7 release

### PR DESCRIPTION
**Pull request by yupeng9**
Part of the 0.7 release prep

### CODE REVIEW

File: NOTICE-binary [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
 // ------------------------------------------------------------------
 // NOTICE file corresponding to the section 4d of The Apache License,
-// Version 2.0, in this case for
+// Version 2.0, in this case for 
 // ------------------------------------------------------------------
 
+Apache Pinot (incubating)
```
Comment (jackjlli) [inline NOTICE-binary:9]: Probably put these lines to the top?

File: LICENSE-binary [hunk 2]
```diff
@@ -206,16 +206,18 @@ License Version 2.0:
...
+com.fasterxml.jackson.core:jackson-core:2.10.0
+com.fasterxml.jackson.core:jackson-databind:2.10.0
+com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.10.0
+com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.10.0
+com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.11.3
+com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:2.10.0
+com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.10.0
+com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.10.0
+com.fasterxml.woodstox:woodstox-core:6.0.1
+om.fasterxml:aalto-xml:1.0.0
```
Comment (fx19880617) [inline LICENSE-binary:219]: ```com.fasterxml:aalto-xml:1.0.0```

File: LICENSE-binary [hunk 3]
```diff
@@ -386,12 +408,16 @@ of these licenses.
...
 args4j:args4j:2.32
-com.azure:azure-core-http-netty:1.3.0
-com.azure:azure-core:1.2.0
-com.azure:azure-storage-blob:12.4.0
-com.azure:azure-storage-common:12.4.0
-com.azure:azure-storage-file-datalake:12.0.0-beta.12
-com.microsoft.azure:azure-data-lake-store-sdk:2.1.5
+com.azure:azure-core-http-netty:1.7.1
+com.azure:azure-core:1.12.0
+om.azure:azure-identity:1.2.2
```
Comment (fx19880617) [inline LICENSE-binary:413]: ```com.azure:azure-identity:1.2.2```

File: LICENSE-binary [hunk 4]
```diff
@@ -451,18 +477,13 @@ org.glassfish.hk2:osgi-resource-locator:1.0.1
...
-
-junit:junit:3.8.1
-
-
 Eclipse Public License (EPL) 1.0
 --------------------------------
 (see licenses-binary/LICENSE-epl-1.0.txt)
 
 org.aspectj:aspectjrt:1.8.4
+om.mchange:c3p0:0.9.5.4
```
Comment (fx19880617) [inline LICENSE-binary:485]: ```com.mchange:c3p0:0.9.5.4```

File: NOTICE-binary (hunk 1 above)
Comment (yupeng9) [reply NOTICE-binary:9]: This is done by following the instructions at https://cwiki.apache.org/confluence/display/PINOT/Creating+an+Apache+Release

**Comment by yupeng9** [comment]
@fx19880617 Comments addressed. PTAL

File: NOTICE-binary (hunk 1 above)
Comment (jackjlli) [reply NOTICE-binary:9]: Yes, I know that. But that doesn't mean I have to follow what the script is generated. Check out other apache projects like spark:
https://github.com/apache/spark/blob/master/NOTICE-binary

Comment (yupeng9) [reply NOTICE-binary:1]: Sure. Updated. PTAL

**Review by jackjlli — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**